guestbook/README.md

```diff
@@ -4,704 +4,354 @@
+{: .note}
 
-<!-- END MUNGE: GENERATED_TOC -->
+{% endcapture %}
 
-The example consists of:
+{% capture objectives %}
+* Start up a redis master.
+* Start up a redis slave.
+* Start up the guestbook frontend.
+* Expose and view the Frontend Service
+* Clean up.
+{% endcapture %}
 
-- A web frontend
-- A [redis](http://redis.io/) master (for storage), and a replicated set of redis 'slaves'.
+{% capture prerequisites %}
 
-The web frontend interacts with the redis master via javascript redis API calls.
+{% include task-tutorial-prereqs.md %}
+Download the following configuration files:
+1. [redis-master-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-master-deployment.yaml)
+2. [redis-master-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-master-service.yaml)
+3. [redis-slave-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-slave-deployment.yaml)
+4. [redis-slave-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-slave-service.yaml)
+5. [frontend-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/frontend-deployment.yaml)
+6. [frontend-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/frontend-service.yaml)
 
-**Note**:  If you are running this example on a [Google Container Engine](https://cloud.google.com/container-engine/) installation, see [this Google Container Engine guestbook walkthrough](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead. The basic concepts are the same, but the walkthrough is tailored to a Container Engine setup.
+{% endcapture %}
 
-### Prerequisites
+{% capture lessoncontent %}
 
-This example requires a running Kubernetes cluster. First, check that kubectl is properly configured by getting the cluster state:
+## Start up the Redis Master
+The guestbook application uses Redis to store its data. It writes its data to a Redis master instance and reads data from multiple Redis worker (slave) instances.
 
-```console
-$ kubectl cluster-info
-```
-
-If you see a url response, you are ready to go. If not, read the [Getting Started guides](http://kubernetes.io/docs/getting-started-guides/) for how to get started, and follow the [prerequisites](http://kubernetes.io/docs/user-guide/prereqs/) to install and configure `kubectl`. As noted above, if you have a Google Container Engine cluster set up, read [this example](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
-
-All the files referenced in this example can be downloaded [from GitHub](https://git.k8s.io/examples/guestbook).
-
-### Quick Start
+### Creating the Redis Master Deployment
+1. `cd` to the folder you saved the `.yaml.` files.
```

Comment:
Rewrite as:

1.  Launch a terminal window in the directory you downloaded the manifest files
2. Apply the Redis Master Deployment to the cluster using the `redis-master-deployment.yaml` file:

ditto for other steps.